mixer/doc/deploy-mixer-with-custom-adapter.md

```diff
@@ -0,0 +1,139 @@
+# Deploying Mixer with custom adapters
+
+Congratulations! You've managed to build a complete Mixer adapter and have validated
+that it works via local testing.
+
+Follow these steps to deploy a Mixer with your custom adapter to your service
+mesh for use.
+
+1. Build a docker image of Mixer with your custom adapter.
+
+   Execute the following command:
+
+   ```bash
+   bazel run //mixer/docker:mixer
```

Comment:
I ran this from both istio and istio/mixer and I am getting
```
guptasu@guptasu:~/go/src/istio.io/istio$ bazel run //mixer/docker:mixer
INFO: Found 1 target...
Target //mixer/docker:mixer up-to-date:
  bazel-bin/mixer/docker/mixer-layer.tar
INFO: Elapsed time: 0.269s, Critical Path: 0.00s

INFO: Running command line: bazel-bin/mixer/docker/mixer
Tagging 33175783ccb2d00882b375e963cfc8e91f2eee79ccde9c0fa09f4bd602e1714c as istio/mixer/docker:mixer
Invalid repository name (mixer/docker), only [a-z0-9-_.] are allowed
ERROR: Non-zero return code '1' from command: Process exited with status 1.
```

Reply:
Are you running this on a Mac?  This command I've validated directly from my goobuntu machine.

Reply:
goobuntu

Reply:
weird. I can't duplicate. which version of docker are you running?